k8s/generate_code.sh

```
@@ -40,24 +40,19 @@ fi
 if ! command -v controller-gen > /dev/null; then
   echo "controller-gen is missing"
   echo "Prepare to install controller-gen"
-  go install sigs.k8s.io/controller-tools/cmd/controller-gen@${CONTROLLER_TOOLS_VERSION}
+  GOFLAGS= go install sigs.k8s.io/controller-tools/cmd/controller-gen@${CONTROLLER_TOOLS_VERSION}
```

Shellcheck (0.10.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)